css/styles.css

```diff
@@ -0,0 +1,53 @@
+header {
+  background-color: #4c2676;
+  color: #fff;
+}
+
+h1 {
+  display: flex;
+  margin: 0;
+  padding: 2.5%;
+  width: 100vw;
+}
+
+body {
+  background-color: #eee;
+}
+
+#image {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
Your `#image` doesn't have any children, so `display: flex`, `align-items:` or `justify-content` isn't doing anything here. It should be removed